far/findfile.cpp

```diff
@@ -3107,8 +3102,8 @@ FindFiles::FindFiles():
 
 		FindAskDlg[FAD_EDIT_MASK].strHistory = L"Masks"sv;
 		FindAskDlg[FAD_EDIT_TEXT].strHistory = L"SearchText"sv;
-		FindAskDlg[FAD_EDIT_HEX].strMask = L"HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH HH"sv;
-		FindAskDlg[FAD_CHECKBOX_NOTCONTAINING].Selected = Options.NotContaining;
+		FindAskDlg[FAD_EDIT_HEX].strMask = HexMask(24);
```

Comment:
Not sure if you noticed, but the original mask was `23` bytes, which I believe was done so that the content fit in the _Containing_ Hex control without scrolling. `24` bytes causes the control to scroll at the very last byte. Just thought I'd point it out.

```suggestion
		FindAskDlg[FAD_EDIT_HEX].strMask = HexMask(23);
```

Reply:
Hm-mm... Yes, I did it intentionally. I thought that 23 was a weird number, so I adjusted it to something more "round." 24 at least is divided by 8. 🙂 The actual purpose of this number has never occurred to me. I'll change it back. Thank you.

Reply:
Done!